### DISCUSSION TITLE
using button6&7 on mouse to switch workspaces

### DISCUSSION THREAD

**Discussion by pmeeke** [i3/i3#5457] [Q&A] [answered]
I have a logitech MX master 3 with a side scroll wheel. I would like to use the side scroll to switch workspaces..
But its not working :( 
I used `xev -event button` to get my button numbers and put the following code into my config.

```
bindsym $mod+Shift+button6 workspace next
bindsym $mod+Shift+button7 workspace prev
```

But no joy. 
What am I doing wrong?

**Comment by orestisfl** [comment]
Have you reloaded your config?

If yes and it still doesn't work, some logs might help https://i3wm.org/docs/debugging.html

**Comment by pmeeke** [comment, marked as answer]
Got it working with..
```
bindsym --whole-window --border $mod+Shift+button6 workspace next
bindsym --whole-window --border $mod+Shift+button7 workspace prev
```

Thanks :)
